File: problemas/iniciante/1072/1072.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        
        int N, X;

        N = Integer.parseInt(in.readLine());

        int incluso = 0;
        for(int i = 0; i < N; ++i){
            X = Integer.parseInt(in.readLine());

            if(10 <= X && X <= 20){
                ++incluso;
            }
        }

        System.out.printf("%d in\n%d out\n", incluso, N - incluso);
    }
}

File: problemas/iniciante/1072/1072.cs
using System;

class URI {
    static void Main(string[] args) {
        int N, X;

        N = int.Parse(Console.ReadLine());

        int incluso = 0;
        for(int i = 0; i < N; ++i){
            X = int.Parse(Console.ReadLine());

            if(10 <= X && X <= 20){
                ++incluso;
            }
        }

        Console.WriteLine($"{incluso} in");
        Console.WriteLine($"{N - incluso} out");
    }
}